ceph.spec.in

```diff
@@ -604,6 +604,11 @@ Group:          System/Filesystems
 Requires:       ceph-mgr = %{_epoch_prefix}%{version}-%{release}
 Requires:       python%{_python_buildid}-remoto
 Requires:       ceph-daemon = %{_epoch_prefix}%{version}-%{release}
+%if 0%{?suse_version}
+Requires:       openssh
+%else
```

Comment:
In the past we've found that using `%else` for distros is problematic (mainly because it's ambiguous). Could you change this to:

```
%if 0%{?suse_version}
Requires: openssh
%endif
%if 0%{?rhel} || 0%{?fedora}
Requires: opensuse-clients
%endif
```

Reply:
fixed